Assets/Engine/Inventory/AttachedContainer.cs

```diff
@@ -126,6 +126,18 @@ public override string ToString()
             return $"{name}({nameof(AttachedContainer)})[size: {container.Size}, items: {container.ItemCount}]";
         }
 
+        public string NameToString()
+        {
+            return $"{name}";
+        }
+
+        /// <summary>
+        /// Verify if this AttachedContainer is attached to a right or to a left hand
+        /// </summary>
+        public bool IsAttachedToHands()
+        {
+            return NameToString() == "hand_r" || NameToString() == "hand_l";
```

Comment:
I like the idea of this method, but I don't think it belongs on the container. Using names for identification is also tricky (think claws for example).

If you were to implement this method on the `Inventory` class, you could access the hands and check if the container is in the list.